fontforge/python.c

```diff
@@ -1596,6 +1604,82 @@ Py_RETURN_NONE;
 return( reto );
 }
 
+int NibCheck(SplineSet *ss);
+
+int PyFF_ConvexNibID(char *tok) {
+   int r = ConvexNibID(tok);
+   if ( r==-1 )
+	PyErr_Format(PyExc_TypeError, "Unrecognized convex nib context name" );
```

Comment:
I haven't checked anywhere else in this file but do we make these translatable elsewhere?

Reply:
There could be some corner of the FF Python API that is I18Ned, but either the vast majority or all of it is not.